frontend/src/metabase/selectors/settings.ts

```diff
@@ -103,6 +103,9 @@ export const getUpgradeUrl = createSelector(
   },
 );
 
+/**
+ * ! The tokenStatus is only visible to admins
```

Comment:
@WiNloSt git says you made this, do you remember if there was a specific reason tokenStatus was used instead of token-features which is public?
It seems that most places that are using this are inside the admin settings so we should be fine, there's only one usage I found that's outside of it: https://github.com/metabase/metabase/blob/957efd97549833357a7cde139f4302c1964a74e7/frontend/src/metabase/admin/tasks/components/Help/Help.tsx#L95

Reply:
I originally used `token-features`
https://github.com/metabase/metabase/blob/31951b43ddc4c4e1f707d32357e738363af27223/frontend/src/metabase/selectors/settings.ts#L51-L55

but that got changed in https://github.com/metabase/metabase/pull/32432/files#diff-531efecfe0b6ca5c45e96de468780cabafe25a4379b1b29b2f2be6f6bede1e3aR104-R107

_screenshot in case GitHub doesn't go to the change_
![image](https://github.com/metabase/metabase/assets/1937582/670dec23-be46-49cb-94b0-0242e6803d23)

Can you ask Alex P. why that's the case?

Reply:
Thanks 🙏 , moved the discussion on slack here: https://metaboat.slack.com/archives/C505ZNNH4/p1708602083391909